tools.yml

```diff
@@ -1019,6 +1023,13 @@ tools:
     mem: 10
   toolshed.g2.bx.psu.edu/repos/iuc/hypo/hypo/.*:
     cores: 8
+  toolshed.g2.bx.psu.edu/repos/bgruening/instagraal/instagraal/.*:
+    cores: 1
+    gpus: 1
+    mem: 30
+    scheduling:
+      accept:
```

Comment:
This tools __needs__ a GPU. Should this then be `required`?

Reply:
Could be, I hope @cat-bro or @nuwang can give some advice here.

Reply:
Specifying gpus should be sufficient.